### DISCUSSION TITLE
How to show summary in output and PR commemt

### DISCUSSION THREAD

**Discussion by jcook-uptycs** [danger/danger-js#1369] [General]
We use danger in a GitHub workflow. Most of our developers when they see a failed check on the PR go directly to the  `details` link. Which takes them to the output of the danger command itself. They do this because all other failed checks this is exactly what they need to do.

However with danger they can't find what they are looking for there.

I would like to be able to have both the output contain the summary from `--text-only` AND have the PR comment add/updated. 

How can this be done, without running danger twice.

**Comment by orta** [comment]
Try use the summary markdown support: https://github.com/danger/danger-js/blob/main/CHANGELOG.md#1110

**Comment by jcook-uptycs** [comment]
That looks like an interesting option. 

Is there anyway to have the whole Danger comment/summary sent there? Or would each rule need to be updated to add support for this output type?

**Reply by orta** [reply]
> . Potentially open to having the main comment in the job summary if someone can make a good case for it in the Danger JS issues

No-one has ever asked for this before, perhaps the API could be `danger.github.setSummaryMarkdown(true)` which would indicate you want that? You're welcome to add that
